File: ProgramingFundamentalsWithJava/LabsAndExercises/Arrays/DayOfWeek.java
import java.util.Scanner;

public class DayOfWeek {
    public static void main(String[] args) {
        int day = new Scanner(System.in).nextInt();
        String[] dayName = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
        if (day>=1&&day<=7){
            System.out.println(dayName[day-1]);
        }else {
            System.out.println("Invalid day!");
                }


            }
        }

File: Javascript Advanced/SyntaxFunctionsAndStatements/DayOfWeek.js
function solve(day){
    let days = {
        'Monday': 1,
        'Tuesday': 2,
        'Wednesday': 3,
        'Thursday': 4,
        'Friday': 5,
        'Saturday': 6,
        'Sunday': 7,
    }
    let result;
    days[day] === undefined ? result = 'error' : result= days[day];
    return result;
}

console.log(solve(''));